### PR TITLE
ref(feature-flag): Enable images-loaded-v2 by default - GA

### PR DESCRIPTION
**Pull request by priscilawebdev**
This sets the `images-loaded-v2` feature flag to true for all users for the ga release

### CODE REVIEW

**Comment by flub** [comment]
https://www.notion.so/sentry/Symbol-and-DIF-Visibility-bc42163d70964cbf8fbbe9b782da1087 more background on what this is here.  There is even a half-baked "Deployment" page which is probably most interesting, if incomplete.

**Review by flub — APPROVED** [review]

**Comment by flub** [comment]
I think the visual snapshot diffs look fine, @priscilawebdev could you check them as well and approve if so?

File: tests/symbolicator/test_payload_full.py
Comment (tonyo) [inline tests/symbolicator/test_payload_full.py:107]: Since we're GA-ing this (basically, making a default), should we also update the tests?
Otherwise we'll be testing not what we're using.

Comment (tonyo) [reply tests/symbolicator/test_payload_full.py:107]: (unless we have tests for both states)

Comment (flub) [reply tests/symbolicator/test_payload_full.py:107]: The file also has a `test_resolving_with_candidates_sentry_source` which explicitly checks this feature.  I was considering updating the snapshots instead of this, but this makes the snapshots less brittle while the feature is still tested.  Which I think was the reasoning we chose when we implemented this the first time round.

Comment (flub) [reply tests/symbolicator/test_payload_full.py:107]: Having the test suite independent of the feature flag also makes it easier to toggle the flag.  There could be an argument that we shouldn't do this in the same PR to make this easier even.  I'll leave that choice up to you as you're most likely to be flipping it back if needed.

Comment (tonyo) [reply tests/symbolicator/test_payload_full.py:107]: > The file also has a test_resolving_with_candidates_sentry_source which explicitly checks this feature.

That's good, but the test coverage for code that matters (i.e. that actually runs in production and on-prem by default) actually dropped, no? 1 test versus N tests that we had with the features disabled.

Comment (flub) [reply tests/symbolicator/test_payload_full.py:107]: When we implemented this we decided there was sufficient test coverage for the feature in symbolicator itself, which tests this more extensively.  The main functionality of the tests here is to ensure that sentry actually switches on this functionality and merges it into the event.  The actual contents of the extra stuff is tested in symbolicator.

**Review by tonyo — APPROVED** [review]

**Comment by flub** [comment]
I'll wait with merging/deploying this till Monday now, but could we have approval of everyone by then please?  /cc @beezz @jan-auer

**Review by beezz — APPROVED** [review]

**Review by jan-auer — APPROVED** [review]

**Pull request merged**
